build/common.props

```diff
@@ -12,7 +12,7 @@
     <PackageTags>aws;sns;sqs</PackageTags>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>$(PackageProjectUrl).git</RepositoryUrl>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I will switch this back on and add the particular warnings for now. We are using methods that have been made obsolete.